business_requirement/README.rst

```diff
@@ -0,0 +1,285 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Business Requirement
+====================
+
+Introduction
+^^^^^^^^^^^^
+
+This module is part of a set ("Business Requirement") and provides the basic 
```

Comment:
I have pending for v8 a remodeling of the intro of the README. Later on we might adapt it to v9 as well